bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/SelectedLayersHandler.js

```diff
@@ -45,8 +45,13 @@ class UIService extends StateHandler {
         const toDataPosition = (layers.length - 1) - toPosition;
         this.sandbox.postRequestByName('RearrangeSelectedMapLayerRequest', [layer.getId(), toDataPosition]);
     }
+
+    changeLayerStyle (layer, styleName) {
+        this.sandbox.postRequestByName('ChangeMapLayerStyleRequest', [layer.getId(), styleName]);
```

Comment:
We should also listen to AfterChangeMapLayerStyleEvent and update the selected value in the dropdown based on it if the style is changed by some other functionality.

Reply:
We are listening to it in `LayerViewTabsStateHandler`.

Reply:
Excellent!